creator-node/src/routes/tracks.js

```diff
@@ -60,7 +60,12 @@ const getFileName = (req) => {
 
 const server = new tus.Server()
 server.datastore = new tus.FileStore({
-  path: tmpTrackArtifactsPath, // has to be a path that exists
+  // Has to be a path that exists
+  path: tmpTrackArtifactsPath,
+  // In the tus-node-server example, they set the file path using the 'path' key. However, in their package,
+  // doing so strips off any forward slashes. The 'directory' key overrides any 'path' specified, so use
+  // this key instead.
+  directory: tmpTrackArtifactsPath,
```

Comment:
does this not end up with /file_storage/file_storage? jw b/c directory + path seems to suggest combining

Reply:
lol one would think https://github.com/tus/tus-node-server/blob/master/lib/stores/FileStore.js#L27

Reply:
yeah, you're right in the terminology that this would be the intended behavior. looking at the source code, it seems like it is not used that way.

FileStore ctor:
```
        super(options);

        this.directory = options.directory || options.path.replace(/^\//, '');

        this.extensions = ['creation', 'creation-defer-length'];
        this.configstore = new Configstore(`${pkg.name}-${pkg.version}`);
        this._checkOrCreateDirectory();
```

DataStore ctor: (the super class)
```
        super();
        if (!options || !options.path) {
            throw new Error('Store must have a path');
        }
        if (options.namingFunction && typeof options.namingFunction !== 'function') {
            throw new Error('namingFunction must be a function');
        }
        this.path = options.path;
        this.generateFileName = options.namingFunction || Uid.rand;
        this.relativeLocation = options.relativeLocation || false;
```

the code will err if we dont pass in a path unfortunately

Reply:
the super class to DataStore is EventEmitter and I dont think that has any explicit path/directory logic

Reply:
kinda hacky / not a fan of this mixed terminology usage but this is the temp solution 🌵

Reply:
also on their package side setting directory to the path value is pretty confusing